sympy/stats/crv_types.py

```diff
@@ -1917,6 +1919,95 @@ def Logistic(name, mu, s):
 
     return rv(name, LogisticDistribution, (mu, s))
 
+#-------------------------------------------------------------------------------
+# Log-logistic distribution --------------------------------------------------------
+
+
+class LogLogisticDistribution(SingleContinuousDistribution):
+    _argnames = ('alpha', 'beta')
+
+    set = Interval(0, oo)
+
+    @staticmethod
+    def check(alpha, beta):
+        _value_check(alpha > 0, "Scale parameter Alpha must be positive.")
+        _value_check(beta > 0, "Shape parameter Beta must be positive.")
+
+    def pdf(self, x):
+        a, b = self.alpha, self.beta
+        return ((b / a) * (x / a)**(b - 1)) / (1 + (x / a)**b)**2
```

Comment:
Spaces are generally included around operators `+` and `-` to distinguish the terms.
The spaces within terms (ie around `*` or `/`) can be skipped.

Reply:
go through [this](https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements).